crates/vault-registry/src/tests.rs

```diff
@@ -1036,9 +1039,13 @@ mod get_vaults_below_premium_collaterlization_tests {
             add_vault(id1.clone(), issue_tokens1, collateral1);
             add_vault(id2.clone(), issue_tokens2, collateral2);
 
+            // set back secure threshold
+            let secure = UnsignedFixedPoint::checked_from_rational(200, 100).unwrap(); // 200%
+            VaultRegistry::_set_secure_collateral_threshold(DEFAULT_CURRENCY_PAIR, secure);
+
             assert_eq!(
-                VaultRegistry::get_premium_redeem_vaults(),
-                Ok(vec![(id1, wrapped(issue_tokens1)), (id2, wrapped(issue_tokens2))])
+                VaultRegistry::get_premium_redeem_vaults(400_u32),
+                Ok(vec![(id2, wrapped(452)), (id1, wrapped(451))])
```

Comment:
If issued_tokens is 50, how come the returned amount is 452??

Reply:
fixed it and updated the mock to pass the correct value.